ci/utils/ubuntu/run_smoke_test.sh

```diff
@@ -43,18 +43,33 @@ verifytdx() {
 
 # Function to create TD image
 createtd() {
+    IMAGE_TYPE=$1
+    echo "Image type is $IMAGE_TYPE"
     echo "$GUEST_IMG_DIR"
     cd "$GUEST_IMG_DIR"
-    
+    if [[ "$BRANCH_NAME" == "noble-24.04" ]]; then
+        image_version=24.04
+    else
+        image_version=24.10
+    fi
     # Check if TD image already exists
     if [ -e td_image_created ]; then
         echo "TD image already present"
     else
-        if [[ "$BRANCH_NAME" == "noble-24.04" ]]; then
-            ./create-td-image.sh -v 24.04
-        else
-            ./create-td-image.sh
-        fi
+        if [ "$IMAGE_TYPE" == "custom" ]; then
+            echo "Inside Custom test"
+	    ./create-td-image.sh -v "$image_version" -o tdx-guest.qcow2 -u tdx_test -p intel123 -n test-guest | tee create_td_image.log
+	    QCOW2_IMG=tdx-guest.qcow2
+            # Check if custom TDX image was created
+	    if grep -viq "tdx-guest.qcow2" create_td_image.log; then
```

Comment:
Code to verify the final image is indeed matching the custom name provided above.